index.html

```diff
@@ -348,6 +504,13 @@ <h5>ART</h5>
                                 <li><a href=""><U class="S">U.S.</U></a></li>
                                 <li><a href="">Politics</a></li>
                                 <li><a href="">Election 2020</a></li>
+                                <li><a href="">Home Page</a></li>
+                                <li><a href="">World</a></li>
+                                <li><a href=""><U class="S">U.S.</U></a></li>
+                                <li><a href="">Politics</a></li>
+                                <li><a href="">Election 2020</a></li>
+                                <li><a href="">Politics</a></li>
+                                <li><a href="">Election 2020</a></li>
```

Comment:
There are a tags with empty href. Remember to use a hash(#) symbol in between the quotes, if there is no specific file path to include.